snippet_parser/core.py

```diff
@@ -271,7 +271,7 @@ def extract(self, wikitext):
                         sr, encoding = 'utf-8', method = 'html')).strip()
                     snippets_in_section.add(snippet)
 
-            sectitle = str(section.get(0).title.strip()) if i != 0 else ''
+            sectitle = str(section.get(0).title.strip_code().strip()) if i != 0 else ''
```

Comment:
```suggestion
            sectitle = ''
            if i != 0:
                # Re-parse the section title because fast_parse is
                # configured to ignore style tags (see above and 
                # https://github.com/earwig/mwparserfromhell/issues/40),
                # but we do want to remove them now with strip_code().
                sectitle = mwparserfromhell.parse(
                    str(section.get(0).title).strip()).strip_code()
```